README.md

```diff
@@ -1 +1,50 @@
-# React-Capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# React Capstone: Studio ghibli Library
+
+> This is a simple project made following instructions while a student in Microverse. You can search for Studio Ghibli movies and click on their cards for a more detailed movie description and information.
+
+
+## Video
+> [Here](https://www.loom.com/share/641f2268ec3343068385abb7ce6d1b1c) is a brief video presentation about the project.
+
+## Built With
+
+- React
+- Redux
+
+
+## Getting Started
+
+- Please clone this repository on your local pc as usual and run `npm i` to install all the necessary libraries;
+- Run `npm start` and feel free to code away.
+
+## Authors
+
+👤 **Thiago Allan Porro Ponce**
+
+- GitHub: [@Thi-Ponce](https://github.com/Thi-Ponce)
+- Twitter: [@Thi_ponce](https://twitter.com/Thi_ponce)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/thiago-ponce)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+If you like this project please let me know!
+
+## Acknowledgments
+
+- Thank you to all the friends that helped me along the way to create this.
+- Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
+
+## See live
+
+[Live project to be added soon](https://github.com/Thi-Ponce/react-mathmagician)
```

Comment:
Kindly deploy the application using Heroku, Netlify, or GitHub Pages and add the deployed link 👍